uavcan/node/432.GetPortInfo.1.0.uavcan

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
This definition has significantly changed from the old DSDL. It should be in v0.1 initially. This should be our top priority to get stabilized.